packages/cli/lib/lib/webpack/run-webpack.js

```diff
@@ -17,22 +17,6 @@ async function devBuild(env) {
 
 	let compiler = webpack(config);
 	return new Promise((res, rej) => {
-		compiler.hooks.emit.tapAsync('CliDevPlugin', (compilation, callback) => {
-			let missingDeps = compilation.missingDependencies;
-			let nodeModulesPath = resolve(__dirname, '../../../node_modules');
-
-			// ...tell webpack to watch node_modules recursively until they appear.
-			if (
-				Array.from(missingDeps).some(
-					file => file.indexOf(nodeModulesPath) !== -1
-				)
-			) {
-				compilation.contextDependencies.push(nodeModulesPath);
-			}
-
-			callback();
-		});
-
```

Comment:
This was adopted from CRA's `WatchMissingNodeModulesPlugin`, however, it seems to have been broken for a few years. This path that it watches resolves to `<project-name>/node_modules/preact-cli/node_modules` which is pretty pointless, as nothing will ever exist/be added to that.

Rather than correct it, I've elected to remove it after seeing [it was recently cut from CRA in v5](https://github.com/facebook/create-react-app/pull/11201). It apparently was having a rather large performance impact which they either couldn't solve or decided it wasn't worth solving. Not entirely sure which.

I figure there's probably no reason to doubt the performance issues claims, so I cut it.